eng/pipelines/templates/jobs/ci.mgmt.yml

```diff
@@ -70,7 +69,11 @@ jobs:
           AgentImage: $(OSVmImage)
       - script: "echo $(system.pullrequest.pullrequestnumber), http://github.com/$(build.repository.id), http://github.com/$(build.repository.ID)"
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
-      - script: "dotnet msbuild mgmt.proj /v:n /t:RunTests /p:Scope=${{ parameters.Scope }} /p:ForPublishing=${{ parameters.ShouldPublish }} /clp:ShowtimeStamp $(RPScopeArgs)"
+      - script: |
+          set ShouldPublish=false
```

Comment:
Why would we need this in the test job?

Reply:
All that the [ForPublishing](https://github.com/Azure/azure-sdk-for-net/blob/c33ced036877429d0e9599431eb915b2dc218471/eng/mgmt.proj#L13) property does is ensure the scope is not empty.  I guess it's not necessary for the test job.

Reply:
Based on my other feedback it probably doesn't matter as we shouldn't need to change ci.mgmt.yml at all.